core/templates/domain/skill/skill-creation-backend-api.service.ts

```diff
@@ -67,20 +72,16 @@ export class SkillCreationBackendApiService {
       successCallback: (value: SkillCreationBackendResponse) => void,
       errorCallback: (reason: string) => void,
       description: string, rubrics: RubricBackendDict, explanation: string,
-      linkedTopicIds: string[], imagesData: ImageData[]): void {
+      linkedTopicIds: string[], files: Record<string, unknown>): void {
```

Comment:
Why is `unknown` here?

Reply:
I took this type from `image local storage service`

Reply:
ditto as above

Reply:
Replied above.